app/assets/templates/flats/flats.html

```diff
@@ -0,0 +1,117 @@
+        </div>
+        <div class="form-group">
+          <label>Is WIFI available?</label>
+          <input type="radio" ng-model="flat.property_have_wifi" value="true">Yes</input>
+          <input type="radio" ng-model="flat.property_have_wifi" value="false">No</input>
+        </div>
+        <div class="form-group">
+          <label>Are pets allowed in the building?</label>
+          <input type="radio" ng-model="flat.are_pets_allowed" value="true">Yes</input>
+          <input type="radio" ng-model="flat.are_pets_allowed" value="false">No</input>
+        </div>
+      </div>
+     
+<!-- room  -->
+      <div class="col-lg-offset-6 col-lg-6 room-panel">
+        <div class="form-group">
+          <label>Rent amount per month</label>
+          <input type="text" class="form-control" placeholder="e.g. 6000" ng-model="flat.room_rent_amount">
+        </div>
+        <div class="form-group">
+          <label>When is the room available?</label>
+          <input type="date" class="form-control" placeholder="Enter content" ng-model="flat.room_date_available">
+        </div>
+        <div class="form-group">
+          <label>Do you have a preference of gender?</label>
+          <input type="radio" ng-model="flat.room_preference" value="male">Male</input>
+          <input type="radio" ng-model="flat.room_preference" value="female">Female</input>
+          <input type="radio" ng-model="flat.room_preference" value="malefemale">Either</input>
+        </div>
+        <div class="form-group">
+          <label>How many beds are provided?</label>
+          <select ng-model="flat.room_no_beds" ng-options="num.Name as num.Name for num in Num track by num.Id">
+           </select>
+        </div>
+        <div class="form-group">
+          <label>What size is/are the bed(s)</label>
+          <select ng-model="flat.room_type_bed" ng-options="bed.Name as bed.Name for bed in Bed track by bed.Id">
+           </select>
+
+        </div>
+        <div class="form-group">
+          <label>Is the room furnished?</label>
+           <input type="radio" ng-model="flat.is_room_furnished" value="true">Yes</input>
+          <input type="radio" ng-model="flat.is_room_furnished" value="false">No</input>
+        </div>
+        <div class="form-group">
+          <label>What type of bathroom is provided?</label>
+          <select ng-model="flat.room_bathroom" ng-options="roomBathroom.Name as roomBathroom.Name for roomBathroom in RoomBathroom track by roomBathroom.Id">
+           </select>
+        </div>
+        <div class="form-group">
+          <label>Provide a general description of the room</label>
+          <input type="text" class="form-control" placeholder="Enter description here" ng-model="flat.room_description">
+        </div>
+      </div>
+      <button type="submit" class="btn btn-default" id="submit-btn">Submit</button>
+    </form>
+  </div>
+</div>
+</section>
+
```

Comment:
please check `scroll_past_end` in your [sublime preferences](http://sublime-text-unofficial-documentation.readthedocs.org/en/latest/reference/settings.html)